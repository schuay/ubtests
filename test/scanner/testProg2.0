i=334aif o=&123then goto hell end;