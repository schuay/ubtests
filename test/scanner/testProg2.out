ident i
=
num 13130
if 
ident o
=
num 123
then 
goto 
ident hell 
end
;